_posts/en/newsletters/2024-06-14-newsletter.md

```diff
@@ -0,0 +1,199 @@
+  this popular LN node implementation.  It includes improvments that
+  help it better work with a pruned full node (see [Newsletter
+  #300][news300 cln prune]), allows the `check` RPC to work with plugins
+  (see [Newsletter #302][news302 cln check]), stability improvements
+  (such as those described in Newsletters [#303][news303 cln chainlag]
+  and [#304][news304 cln feemultiplier]), allows more robust delivery of
+  [offer][topic offers] invoices (see [Newsletter #304][news304 cln
+  offers]), and a fix for a fee overpayment issue when the
+  `ignore_fee_limits` configuration option is used (see [Newsletter
+  #306][news306 cln overpay]).
+
+- [Bitcoin Core 27.1][] is a maintenance release of the predominant
+  full node implementation.  It contains multiple bug fixes.
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #29496][] bumps `TX_MAX_STANDARD_VERSION` to 3, which makes
+  Topologically Restricted Until Confirmation ([TRUC][topic v3 transaction
+  relay]) transactions standard. If a transaction’s version is 3, it will be
+  treated as a TRUC transaction as defined in the BIPs #431 specification. The
+  `CURRENT_VERSION` remains 2, meaning the wallet will not create TRUC
+  transactions yet.
+
+- [Bitcoin Core #28307][] fixes a bug that imposed the 520-byte P2SH maximum
+  script size limit on SegWit redeem scripts for both P2SH-segwit and bech32.
+  This fix enables the creation of multisig [output descriptors][topic
+  descriptors] involving more than 15 keys (up to 20), also allows for the
+  signing of these scripts, along with other post-segwit redeem scripts that
+  surpass the P2SH limit.
+
+- [Bitcoin Core #30047][] refactors the model of the bech32 encoding scheme
+  `charlimit` from a constant of 90 to an `Enum`. This change allows for easy
+  support of new address types that use the bech32 encoding scheme but don't
+  have the same character limit as segwit addresses. For example, to enable
+  parsing [silent payment][topic silent payments] addresses, which require up to
+  118 characters.
+
+- [Bitcoin Core #28979][] updates the behavior of the `sendall` RPC command to
+  document and test the spending of unconfirmed change, and if specified, the
+  spending of all unconfirmed inputs. See Newsletter #198 for more information
+  on the `sendall` command. <!-- If I understand correctly, this PR is about the documentation and testing of an already existing feature.-->
```

Comment:
The documentation of `sendall` was inaccurate in that it said that it would only spend confirmed UTXOs, but Ishaana discovered that it does spend unconfirmed change. Since it does spend some unconfirmed UTXOs, she added support for ancestor aware funding behavior to `sendall` which was missing. She did not change the behavior to allow unconfirmed change, that was already there.